ert_shared/tracker/evaluator.py

```diff
@@ -266,13 +274,7 @@ def _clear_work_queue(self):
             pass
 
     def request_termination(self):
+        config = load_config()
+        wait_for_ws(config.get("url"))
```

Comment:
We discussed briefly that this will fail if the evaluator fails after startup, but before cancellation. Then this will wait forever.

Do we care about that? If so, any ideas on how to fix it?

Reply:
Perhaps I should start writing a test for this. Then we can discuss if we need to care about it